src/main/java/org/sagebionetworks/bridge/exporter/config/SpringConfig.java

```diff
@@ -213,4 +217,10 @@ public ExecutorService workerExecutorService() {
 
         return ImmutableList.copyOf(value);
     }
+
+    @Bean(name = "ddbClientScan")
+    public AmazonDynamoDBClient ddbClientScan() {
+        return new AmazonDynamoDBClient(new ProfileCredentialsProvider());
+    }
```

Comment:
Why do we need a DDB client? Why are we using the ProfileCredentialsProvider instead of the default one?